design/proposals/load_policy/load_policy_validation.md

```diff
@@ -0,0 +1,212 @@
+* [Issue Description](#issue-description)
+* [Solution](#solution)
+   + [Solution - scope](#solution---scope)
+   + [User Interface](#user-interface)
+* [Design](#design)
+   + [Flow Diagrams](#flow-diagrams)
+   + [Sequence Diagrams](#sequence-diagrams)
+      - [Sequence Diagrams - additional info](#sequence-diagrams---additional-info)
+* [Performance](#performance)
+* [Backwards Compatibility](#backwards-compatibility)
+* [Affected Components](#affected-components)
+* [Test Plan](#test-plan)
+   + [Out of Scope](#out-of-scope)
+   + [Prerequisites](#prerequisites)
+* [Documentation](#documentation)
+* [Security](#security)
+* [Infrastructure](#infrastructure)
+* [Audit](#audit)
+* [Open Questions](#open-questions)
+* [Delivery Plan](#delivery-plan)
+   + [Total 1st milestone EE 45](#total-1st-milestone-ee-45)
+      - [Ramp-up EE 2](#ramp-up-ee-2)
+      - [Designs EE 2](#designs-ee-2)
+      - [Implementations EE 19](#implementations-ee-19)
+      - [Testing EE 13](#testing-ee-13)
+      - [Security EE 5](#security-ee-5)
+      - [Docs EE 3](#docs-ee-3)
+      - [Demo EE 1](#demo-ee-1)
+* [Definition of Done](#definition-of-done)
+* [Solution Review](#solution-review)
+
+## Glossary
+TBD
+
+## Useful Links
+- [Feature doc](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-8566)
+- [UX research page](https://ca-il-confluence.il.cyber-ark.com.mcas.ms/display/rndp/JWT+Authenticator+-+load+policy+validation)
+- [JWT authenticator stage 1.1 feature doc](https://ca-il-confluence.il.cyber-ark.com.mcas.ms/display/rndp/JWT+Authenticator+-+re-scope#JWTAuthenticatorrescope-Stage1.1-SimplegenericJWTauthenticatorwithtokenschema)
+
+
+## Issue Description
+In misconfiguration errors, for example:
+hosts with empty JWT annotation value, we are failing at JWT authentication request which is bad UX.
+
+## Solution
+For better UX we want to perform policy validation steps in order to identify configuration errors early as possible. 
+
+### Solution - scope
+This solution is creating a generic infrastructure for all policy entities validation.
+And focusing on Host with JWT authenticator annotations as first use case to implement on this new suggested infrastructure.
+
+### User Interface
+TBD
+
+## Design
+The following diagrams, represent a new policy validation layer which will be triggered as part of the load policy request,
+As the final state after the current logic.
+
+
+### Flow Diagrams
+In our use case, Host with JWT authenticator annotations validation:
+1. Conjur admin define JWT authenticator 
+2. Conjur admin load a policy of Host to be authenticated with JWT authenticator
+3. New policy validation layer is triggered and validate if the host is configured correctly 
+4. Host authenticate using JWT authenticator, due to above validation there are no configuration errors at the authenticate request stage
+
+### Sequence Diagrams
+![Load policy validation sequence diagram](load-policy-validation-sequence-diagram.png)
```

Comment:
Cannot open sequence diagram. Also - can you please send me links to class diagram ?

Reply:
Push the link in the PR description